lib/rules/no-useless-constructor.js

```diff
@@ -25,13 +25,20 @@ module.exports = function(context) {
             return false;
         }
 
-
         return body[0].expression.arguments.every(function(arg, index) {
-            return (arg.type === "Identifier" && arg.name === ctorParams[index].name) ||
```

Comment:
I meant I can update this one lol :)

I'm just trying to say I'd rather you follow-up after this patch is merged if this looks okay to you.